drracket/drracket/tool-lib.rkt

```diff
@@ -238,8 +238,8 @@ all of the names in the tools library, for use defining keybindings
   @{Registers a pair of procedures with DrRacket's online expansion machinery. 
     (See also @racket[drracket:module-language-tools:add-online-expansion-monitor].)
     
-    The first two arguments name a procedure in a module that is loaded by 
-    @racket[dynamic-require] is specially designed separate @racket[place].
+    The procedure @racket[id] from @racket[mod-path] is loaded by 
```

Comment:
This needs to be @racket[_id] to get the right typesetting.

Reply:
```suggestion
    The procedure @racket[_id] from @racket[mod-path] is loaded by
```